src/controllers/user/display.js

```diff
@@ -24,7 +24,8 @@ define(["displaySettings", "userSettingsBuilder", "userSettings"], function (Dis
                     element: view.querySelector(".settingsContainer"),
                     userSettings: userSettings,
                     enableSaveButton: false,
-                    enableSaveConfirmation: false
+                    enableSaveConfirmation: false,
+                    autoFocus: autoFocuser.isEnabled()
```

Comment:
why do we need this?

Reply:
Otherwise, it will focus on non-TV. Initially it was without `autoFocus`, and it is currently in sync with `AutoFocuser` which is enabled on TV only.

Reply:
oh, so `autoFocus` is an already existing value?

Reply:
Yes, it has not been used here before.
https://github.com/jellyfin/jellyfin-web/blob/c10f3feb3149d761c993a0c3b007c09c8a848f14/src/components/displaysettings/displaysettings.js#L295-L297